org.freedesktop.fwupd.json

```diff
@@ -0,0 +1,211 @@
+        "--filesystem=/boot",
+        "--filesystem=/sys",
+        "--filesystem=xdg-download",
+        "--share=network",
+        "--system-talk-name=org.freedesktop.fwupd",
+        "--system-talk-name=org.freedesktop.UPower"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g"
+    },
+    "cleanup": [
+        "*.a",
+        "*.la",
+        "/include",
+        "/lib/girepository-1.0",
+        "/lib/pkgconfig",
+        "/share/bash-completion",
+        "/share/dbus-1/system-services",
+        "/share/gir-1.0",
+        "/share/gtk-doc",
+        "/share/info",
+        "/share/man",
+        "/share/pkgconfig" ],
+    "modules": [
+        {
+          /* not using shared-modules as we need gudev */
+          "name": "udev",
+          "rm-configure": true,
+          "config-opts": [
+            "--disable-hwdb",
+            "--disable-logging",
+            "--disable-introspection",
+            "--disable-keymap",
+            "--disable-mtd_probe"
+          ],
+          "cleanup": [
+            "/etc/udev",
+            "/libexec/*",
+            "/share/gtk-doc/html/libudev/",
+            "/sbin/udevadm"
+          ],
+          "sources": [
+            {
+                "type": "archive",
+                "url": "http://kernel.org/pub/linux/utils/kernel/hotplug/udev-175.tar.bz2",
+                "sha256": "4c7937fe5a1521316ea571188745b9a00a9fdf314228cffc53a7ba9e5968b7ab"
+            },
+            {
+                "type": "script",
+                "dest-filename": "autogen.sh",
+                "commands": [
+                    "autoreconf -vfi"
+                ]
+            }
+          ]
+        },
+        {
+            "name": "libusb",
+            "config-opts": [
+                "--disable-static"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/libusb/libusb/releases/download/v1.0.22/libusb-1.0.22.tar.bz2",
+                    "sha256": "75aeb9d59a4fdb800d329a545c2e6799f732362193b465ea198f2aa275518157"
+                }
+            ]
+        },
+        {
+            "name": "gusb",
+            "buildsystem": "meson",
+            "config-opts": ["-Ddocs=false",
+                            "-Dvapi=false",
+                            "-Dtests=false"],
+            "cleanup": [
+                "/bin/gusbcmd"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://people.freedesktop.org/~hughsient/releases/libgusb-0.3.0.tar.xz",
+                    "sha256": "d8e7950f99b6ae4c3e9b8c65f3692b9635289e6cff8de40c4af41b2e9b348edc"
+                }
+            ]
+        },
+        {
+            "name": "efivar",
+            "buildsystem": "simple",
+            "build-commands": ["make prefix=/app libdir=/app/lib", "make install prefix=/app libdir=/app/lib"],
```

Comment:
Same build comment as below. `make-args`, `make-install-args`.

Reply:
That doesn't work.